extensions/trello/src/TrelloListItem.tsx

```diff
@@ -0,0 +1,40 @@
+import {
+    List,
+    Icon,
+    ActionPanel,
+    OpenInBrowserAction,
+  } from '@raycast/api'
+  import { TrelloResultModel } from './trelloResponse.model'
+  
+  interface TodoListItemProps {
+    result: TrelloResultModel
+  }
+  
+  export const TodoListItem = ({
+    result,
+  }: TodoListItemProps): JSX.Element => {
+    const todo = result
+  
+    return (
+      <List.Item
+        id={todo.id}
+        title={todo.name}
+        // accessoryTitle={todo.due?.toLocaleDateString()}
```

Comment:
How do you use a function like this when the value could sometimes be empty?

Reply:
```typescript
id = {todo?.id ?? "Fallblack Text"}
```

Reply:
@Ali-Parandeh Hmm, but does that apply here? To be more specific I mean `{todo.due.toLocaleDateString()}`, the `.toLocaleDateString()` sometimes results in error if there is no value, i.e. "TypeError: todo.due?.toLocaleDateString is not a function"

Reply:
I cannot get this to work, it's optional right now anyway

Reply:
Is the `due` a Date object?

Reply:
@thomaspaulmann well I made it one in the 'model', but the Trello API returns a string, maybe I was doing something wrong there

Reply:
That could be the case. It depends on what it returns. Best to use string and convert it if you need to.